playbook/app/pb_kits/playbook/pb_date_time_stacked/_date_time_stacked.jsx

```diff
@@ -1,43 +1,46 @@
+
 /* @flow */
 
 import React from 'react'
-import { DateStacked, Flex, FlexItem, TimeStacked } from '../'
-import { buildDataProps } from '../utilities/props'
-import { globalProps } from '../utilities/globalProps.js'
+import { DateStacked, Flex, FlexItem, SectionSeparator, TimeStacked } from '../'
 
 type DateTimeStackedProps = {
   data?: string,
   id?: string,
-  date: date,
+  datetime: string,
```

Comment:
Have we verified that Nitro is not using this? This appears to be a breaking change.
I'm going to add the `breaking change` label until we know otherwise, just to be safe.

Reply:
@evalouderback one thing I was thinking here was whether we could instead deprecate `date` and make `datetime` not required until the next major release. @jasoncypret what might make the most if we go this route is to generate a RW ticket in our backlog to circle back to this kit and remove the deprecation later.

Reply:
@evalouderback [here is how you would do that in React kits](https://github.com/powerhome/playbook/blob/854c06da45f0ad05c8dce57aa1a9f32c3a3a34f8/playbook/app/pb_kits/playbook/pb_time_stacked/_time_stacked.jsx#L24)

Reply:
And in Rails all you do is add:


```rb
prop :date, deprecated: true
```

Reply:
With the above examples, I would propose to keep `date` prop as deprecated remove the `required: true` from `datetime`. Anywhere in your `html.erb`, you will want to do:

```erb
<%= datetime || date %>
```

which serves two purposes:

1. Defaulting to the old prop in areas that are using older kit implementations (current)
2. Self-documents by adding droplets of wet code needing cleanup which will point contributors to the `.rb` and `.jsx` source code files which further document the deprecation itself (along with warning command line output in both the browser and terminal).

Reply:
It's not being used in Nitro in rails or react. Do we still want to make those changes? @thestephenmarshall @jasperfurniss

Reply:
I vote yes bc Playbook is a public repo and breaking changes mean a major `9.x` release which we aren't ready for yet.